Resources/Prototypes/Entities/Objects/Consumable/Drinks/drinks_cups.yml

```diff
@@ -365,3 +365,31 @@
     - HEAD
     quickEquip: false
     sprite: Clothing/Head/Hats/party_water_cup.rsi
+
+- type: entity
+  parent: DrinkBaseCup
+  id: DrinkHotMoffee
+  name: moffee
+  description: It's actually short for "mock coffee," but no one means that when they say it.
+  components:
+  - type: SolutionContainerManager
+    solutions:
+      drink:
+        reagents:
+        - ReagentId: Moffee
+          Quantity: 20
+  - type: Icon
+    sprite: Objects/Consumable/Drinks/hot_coffee.rsi
+    state: icon-vend
+  - type: Sprite
+    sprite: Objects/Consumable/Drinks/hot_coffee.rsi
```

Comment:
You've added hot_moffee.rsi sprites but you're reusing the coffee sprites for the drink here.